src/peft/tuners/tuners_utils.py

```diff
@@ -32,6 +35,47 @@
 logger = logging.getLogger(__name__)
 
 
+@contextmanager
+def onload_layer(layer):
```

Comment:
Could we have a short docstring that explains what is happening here.

Reply:
Added a docstring